### DISCUSSION TITLE
Add guidance on how to set DataClassification on fields

### DISCUSSION THREAD

**Discussion by StefanMaron** [microsoft/alguidelines#165] [Ideas]
I am not sure where the correct place would be, here or on the ms docs site:
https://docs.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/properties/devenv-dataclassification-property

But the current explanation on when to use which value as data classification is (at least to me) not clear at all.
I would like to see some examples for all of those values and a more detailed description.
Maybe this is also a matter of being native English speaker or not, I don't know.

**Comment by JeremyVyska** [comment]
@JesperSchulz - is there any guidance that could be turned into the start of that document?
